README.md

```diff
@@ -97,11 +97,11 @@ with  @org.virtuslab.ash.annotation.SerializabilityTrait.
 
 
 The compiler plugin only checks the classes in the sbt modules where `AkkaSerializationHelperPlugin` is explicitly enabled.
-It may happen that your base trait (like `MySerializable` in the example) may live in an sbt module like `core` where you do **not** want to enable the plugin
-(e.g. for compilation performance reasons).
-You still need to reference `org.virtuslab.ash.SerializabilityTrait` to annotate `MySerializable`, however.
+It may happen that the base trait (like `MySerializable` in the example) lives in a sbt module like `core` where the plugin
+should **not** be enabled (e.g. for compilation performance reasons).
+However, `MySerializable` needs to be annotated with a `org.virtuslab.ash.SerializabilityTrait`.
 In order to have access to the `SerializabilityTrait` annotation without enabling the entire suite of compiler plugins,
-add `AkkaSerializationHelperPlugin.annotation` to `libraryDependencies`:
```

Comment:
Imperative is okay! probably even better than passive voice, which sometimes sounds too formal/stiff

Reply:
So... pls revert here